python-qa-tools/deps/prereq.py

```diff
@@ -0,0 +1,35 @@
+import os, subprocess
```

Comment:
let's replace this with bash script. It would be neat to have bash scripts for:

- Installation (ex: `./instalation.sh`)
- Setup ENV (ex: `./env.sh`)
- Start a chain (ex: ./start_chain.sh --num_nodes 4)
- Tests:
   - `./test_all_txs_queries.sh`
   - `./loadtest.sh`
   - `./multisend_loadtest.sh`
   NOTE: internally these test scripts will call the python scripts as required.
  
Or a Makefile definition would be great instead of executing scripts directly.